docs/DevGuide/ProfilingWithProjections.md

```diff
@@ -21,104 +21,17 @@ means compiling Charm++ with the `--with-production` flag. To enable trace
 collecting you must build with the `--enable-tracing` flag as well. For
 example, on a multicore 64-bit Linux machine the build command would be
 ``` shell
-./build charm++ multicore-linux64 gcc -j8 --with-production --enable-tracing
+./build charm++ multicore-linux-x86_64 gcc -j8 --with-production \
```

Comment:
In `Installation.md` (and I think in `Dockerfile.buildenv`) there are several places that still say `multicore-linux64`. Should these also be updated to say `multicore-linux-x86_64` ?